### ISSUE TITLE
Pull Request Template Formatting Error

### ISSUE THREAD

**Issue by Jon-Corey** [SparkDevNetwork/Rock#5495]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

The pull request template has a formatting error that is causing everything between the Proposed Changes header and the Documentation header to render incorrectly.

What the pull request template currently looks like:

![Pull Request Template with a formatting error](https://github.com/SparkDevNetwork/Rock/assets/110615344/952f03f0-961f-4173-a246-f9e5086379d0)

Here is what the pull request template should look like:

![Pull Request Template without a formatting error](https://github.com/SparkDevNetwork/Rock/assets/110615344/28a08c66-4e68-460b-9c8e-e4465ed6aef2)

### Expected Behavior

The pull request template should render the markdown correctly between the Proposed Changes heading and the Documentation heading.

### Actual Behavior

The pull request template has a blob of unrendered markdown between the Proposed Changes heading and the Documentation heading.

### Steps to Reproduce

1. Go to SparkDevNetwork/Rock in GitHub
2. Go to the Pull requests tab
3. Click on "New pull request"
4. Select a branch other than develop as the compare branch (so that there are changes between the two branches)
5. Click "Create pull request"
6. Select the "Preview" tab on the text entry box
7. Notice the formatting error

### Rock Version

N/A

### Client Culture Setting

N/A
